app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -67,13 +67,17 @@ def prepare_saml_logout_response
     @saml_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse])
   end
 
+  def slo_session
```

Comment:
what does `slo` mean? would it make sense to use the full word rather than an acronym? (if that is in fact what it is..)